app/src/main/java/com/github/se/studybuddies/database/DatabaseConnection.kt

```diff
@@ -389,7 +378,7 @@ class DatabaseConnection : DbRepository {
       val picture = Uri.parse(document.getString("picture") ?: "")
       val members = document.get("members") as List<String>
       val timerStateMap = document.get("timerState") as? Map<String, Any>
-      val endTime = timerStateMap?.get("endTime") as? Long ?: System.currentTimeMillis()
+      val endTime = timerStateMap?.get("endTime") as? Long ?: 0L
       val isRunning = timerStateMap?.get("isRunning") as? Boolean ?: false
```

Comment:
Again it's will be great to have const, so you could reuse the timerState defined in updateGroupTimer